src/Kernel32.Desktop/Kernel32+FindFirstFileExFlags.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+
+    /// <content>
+    /// Contains the <see cref="FindFirstFileExFlags"/> nested enum.
+    /// </content>
+    public partial class Kernel32
+    {
+        /// <summary>
+        /// Optional flags to pass to the <see cref="FindFirstFileEx"/> method.
+        /// </summary>
+        [Flags]
+        public enum FindFirstFileExFlags
+        {
+            /// <summary>
+            /// Searches are case-sensitive.
+            /// </summary>
+            CaseSensitive,
```

Comment:
I think the default value here is 0,but based on [msdn](https://msdn.microsoft.com/en-us/library/windows/desktop/aa364419.aspx) `FindFirstFileExFlags.CaseSensitive` should be 1